unstructured/ingest/interfaces.py

```diff
@@ -654,22 +654,34 @@ def __init__(self, write_config: WriteConfig, connector_config: BaseConnectorCon
         self.write_config = write_config
         self.connector_config = connector_config
 
+    def conform_dict(self, data: dict) -> None:
+        return
+
     @abstractmethod
     def initialize(self):
         """Initializes the connector. Should also validate the connector is properly
         configured."""
 
-    @abstractmethod
     def write(self, docs: t.List[BaseSingleIngestDoc]) -> None:
+        elements_dict = self.get_elements_dict(docs=docs)
+        self.write_raw_dict(elements_dict=elements_dict)
+
+    @abstractmethod
+    def get_elements_dict(self, docs: t.List[BaseSingleIngestDoc]) -> t.List[t.Dict[str, t.Any]]:
         pass
 
     @abstractmethod
     def write_dict(self, *args, elements_dict: t.List[t.Dict[str, t.Any]], **kwargs) -> None:
         pass
 
+    def write_raw_dict(self, *args, elements_dict: t.List[t.Dict[str, t.Any]], **kwargs) -> None:
```

Comment:
I'd rather name this as `write_conformed_dict`, raw sounds as not processed / formatted

Reply:
@rbiseck3 I do see your point, but I think I agree with @ahmetmeleq here. at a glance I immediately know what to expect from `normalize_and_write_dict` whereas I might not have assumed `write_raw_dict` does this without looking at the code.

Reply:
ahh, looks like you just bumped